deltachat-ios/DC/events.swift

```diff
@@ -13,6 +13,13 @@ let dcNotificationContactChanged = Notification.Name(rawValue: "MrEventContactsC
 @_silgen_name("callbackSwift")
 
 public func callbackSwift(event: CInt, data1: CUnsignedLong, data2: CUnsignedLong, data1String: UnsafePointer<Int8>, data2String: UnsafePointer<Int8>) -> UnsafePointer<Int8>? {
+    if event >= DC_EVENT_ERROR && event <= 499 {
```

Comment:
I guess the idea here is to suppress the error events that are in the range of 400 - 499 to be shown to the user, right? 
This condition includes not only DC_EVENT_ERROR_NETWORK, but also also DC_EVENT_ERROR_SELF_NOT_IN_GROUP. Is that ok?

Reply:
yip, that is okay for now. it has to be settled in deltachat.h, however, i do not want to wait for that here.

Reply:
and: the errors are not suppressed.
they are logged as before logger.error("event: \(s)") - and moreover, the AppDelegate.lastErrorString is set for _all_ errors (was not true for DC_EVENT_ERROR_NETWORK before), this was one of the reasons for #328